core/src/main/java/com/datastax/oss/driver/api/core/tracker/RequestTracker.java

```diff
@@ -36,6 +36,33 @@
  */
 public interface RequestTracker extends AutoCloseable {
 
+  /**
+   * Invoked each time new request is created.
+   *
+   * @param request the request to be executed
+   * @param executionProfile the execution profile of this request
+   * @param requestLogPrefix the dedicated log prefix for this request
+   */
+  default void onRequestStart(
```

Comment:
`onRequestCreated` may be a better name as it seems more clear that its at the very beginning.

Reply:
Changed.